style/js/shield_defs.js

```diff
@@ -2529,6 +2529,20 @@ export function loadShields(shieldImages) {
     Color.shields.white
   );
 
+  // Belgium
+  shields["BE:A-road"] = roundedRectShield(
+    Color.shields.white,
+    Color.shields.black
+  );
+  shields["BE:N-road"] = roundedRectShield(
+    Color.shields.blue,
+    Color.shields.white
+  );
+  shields["BE:R-road"] = roundedRectShield(
+    Color.shields.white,
+    Color.shields.black
+  );
+
```

Comment:
Let's keep this section sorted by country code. So `be` would go between `ba` and `bg`.

Reply:
My bad, fixed in force-push.